src/pages/stage-page/StagePage.tsx

```diff
@@ -49,35 +48,33 @@ export const StagePage = ({
       <DateInput
         value={startEvent.eventDate}
         label={endEvent ? "Start Date" : "Date"}
-        name={`${startEvent}-start-date`}
+        name={`${title.replace(" ","-")}-start-date`}
```

Comment:
`title.replace(" ","-")` is repeated in a few places in this page maybe its worth pulling in a "stageID" const?

Reply:
I dont see a stageID anywhere?